roadmap/implementers-guide/src/node/overseer.md

```diff
@@ -54,11 +54,14 @@ The overseer's logic can be described with these functions:
 
 ## On Subsystem Failure
 
-Subsystems are essential tasks meant to run as long as the node does. Subsystems can spawn ephemeral work in the form of jobs, but the subsystems themselves should not go down. If a subsystem goes down, it will be because of a critical error that should take the entire node down as well.
+Subsystems are essential tasks meant to run as long as the node does.The subsystems themselves should not go down. If a subsystem goes down, it will be because of a critical error that should take the entire node down as well.
```

Comment:
```suggestion
Subsystems are essential tasks meant to run as long as the node does. The subsystems themselves should not go down. If a subsystem goes down, it will be because of a critical error that should take the entire node down as well.
```